.travis.yml

```diff
@@ -32,6 +32,7 @@ matrix:
        - TAGS="-tags travis"
        - COVERAGE="-race"
        - GO111MODULE="on"
+       - GOPROXY="https://athens.azurefd.net"
```

Comment:
@sbinet, Why do we need a proxy server in travis CI?

Reply:
This isn't for Travis only, per se, but to test modules and the expected new package delivery system:
https://blog.golang.org/modules2019

Reply:
okay, Thanks. So, are these mirrors paid? Also, I just learned that these could be a victim to man in middle attack. What are your thoughts on this?

Reply:
athens is supposed to be a mirror operated by the community.
there will be bills to be paid, for sure, but I think the gophers won't have to give a credit card upfront.

https should address most of the MiM attacks (all?).
I am not a security expert, though.